audb/core/load.py

```diff
@@ -477,6 +475,84 @@ def job(table: str):
     audeer.rmdir(db_root_tmp)
 
 
+def _load_attachments(
+        attachments: typing.Sequence[str],
+        backend: audbackend.Backend,
+        db_root: str,
+        db: audformat.Database,
+        version: str,
+        cached_versions: typing.Optional[CachedVersions],
+        deps: Dependencies,
+        flavor: Flavor,
+        cache_root: str,
+        num_workers: int,
+        verbose: bool,
+) -> typing.Optional[CachedVersions]:
+    r"""Load attachments to cache.
+
+    Args:
+        attachments: list of attachment IDs
+        backend: backend object
+        db_root: database root
+        db: database object
+        version: database version
+        cached_versions: object representing cached versions
+            of the database
+        deps: database dependency object
+        flavor: database flavor object
+        cache_root: root path of cache
+        num_workers: number of workers to use
+        verbose: if ``True`` show progress bars
+            for each step
+
+    Returns:
+        cached versions object
+            if other versions of the database are found in cache
+
+    """
+    missing_attachments = []
+    for attachment in attachments:
+        path = db.attachments[attachment].path
+        path = audeer.path(db_root, path)
+        if not os.path.exists(path):
```

Comment:
With the proposed changes it is now much easier to find out if an attachment hast been loaded or not - we only have to check if the path of the attachment exists.